devdoc/doublylinkedlist_requirements.md

```diff
@@ -20,13 +20,17 @@ typedef struct DLIST_ENTRY_TAG
     struct DLIST_ENTRY_TAG *Blink;
 } DLIST_ENTRY, *PDLIST_ENTRY;
 
+typedef int (*DLIST_ACTION_FUNCTION)(PDLIST_ENTRY listEntry, void* actionContext, bool* continueProcessing);
+
 extern void DList_InitializeListHead(PDLIST_ENTRY listHead);
```

Comment:
>extern [](http://example.com/codeflow?start=0&length=6)

all C function have by default "extern" linkage, so it can be omitted.

./consider

Reply:
copy pasting from header in next iteration.